htdocs/fourn/class/fournisseur.facture.class.php

```diff
@@ -2481,6 +2481,38 @@ public function insert($notrigger=0)
             $this->db->rollback();
             return -2;
         }
-    }
-}
+    } 
+            /**
+     *  Mise a jour de l'objet ligne de commande en base
+     *
+     *  @return		int		<0 si ko, >0 si ok
+     */
+    function update_total()
```

Comment:
Eldy this function exist in all the element (facture, order, ... customer and fournish) except for fournish bill. I use it to recalc the détail of the line when i change it in one of my module (customline). 
My goal with this PR is to standardise all the function and have the same code for all the element